Formula/omniorb.rb

```diff
@@ -30,17 +30,20 @@ class Omniorb < Formula
   end
 
   def install
+    ENV["PYTHON"] = Formula["python@3.9"].opt_bin/"python3"
```

Comment:
Can we not hardcode `python@3.9` here? That just causes problems when the `depends_on` line is changed.

Please also rebase (not `merge`) to get rid of the merge commit -- @BrewTestBot cannot merge PRs with merge commits. Squash all your commits together when you do so, and use the commit message
```
omniorb: use python3 for bindings
```
or similar. See the commit style guide in the PR template for details.

Reply:
done